plugins/com.google.cloud.tools.eclipse.appengine.validation/xslt/servlet.xsl

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	This stylesheet changes the servlet in web.xml to Servlet 2.5.
+-->
+<xsl:stylesheet version="1.0"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:ns1="http://xmlns.jcp.org/xml/ns/javaee"
+	xmlns:ns2="http://java.sun.com/xml/ns/javaee"
+	exclude-result-prefixes="ns1 ns2">
+
```

Comment:
Try adding

<xsl:output method="xml" indent="yes"/>

as an extra top-level element to pretty print the output a little more.